sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -752,7 +754,7 @@ export interface FeedOptions extends SharedOptions {
     forceQueryPlan?: boolean;
     maxDegreeOfParallelism?: number;
     maxItemCount?: number;
-    partitionKey?: any;
+    partitionKey?: PartitionKey;
```

Comment:
This is also a breaking change, unfortunately

Reply:
The same previous logic for backward compatibility of `partitonKey` applies here as well.